pyomo/common/config.py

```diff
@@ -1901,10 +1895,3 @@ def _data_collector(self, level, prefix, visibility=None, docMode=False):
 
 # Backwards compatibility: ConfigDict was originally named ConfigBlock.
 ConfigBlock = ConfigDict
-
-# In Python3, the items(), etc methods of dict-like things return
-# generator-like objects.
-if six.PY3:
-    ConfigDict.keys = ConfigDict.iterkeys
-    ConfigDict.values = ConfigDict.itervalues
-    ConfigDict.items = ConfigDict.iteritems
```

Comment:
If you are going to remove this, you also need to rename the `iter*` methods to be `items`, `keys,  and `values` and deprecate the `iter*` methods.